### DISCUSSION TITLE
Margin always right in twoside

### DISCUSSION THREAD

**Discussion by martinunland** [fmarotta/kaobook#182] [Q&A] [answered]
Hi there, thank you for your work!
Is there currently a possibility to have the margin always on the right side with the twoside option = true?
Martin

**Comment by fmarotta** [comment, marked as answer]
Hi, thanks for your question. Right now, this is not possible. But besides the margin position, the only thing that changes between even and odd pages in twoside mode is the header, right? So I think the easiest thing would be to use `twoside=false` and then modify the header. You would need to check whether the page is even or odd, and typeset the header accordingly. Please create an issue if you need further help :)
